tools/serviced-storage/driver.go

```diff
@@ -176,6 +186,118 @@ func (c *DriverList) Execute(args []string) error {
 	return nil
 }
 
+// Execute displays orphaned volumes
+func (c *CheckOrphans) Execute(args []string) error {
+	App.initializeLogging()
+	type Metadata struct {
+		CurrentDevice string
+		Snapshots     interface{}
+	}
+
+	directory := GetDefaultDriver(string(c.Args.Path))
```

Comment:
Since this is a devicemapper only function, you should ensure that the FSType is devicemapper and error on other device types.  It may even make more sense to cast the driver and volume as a DeviceMapperDriver and DeviceMapperVolume, can reference the Metadata directories from the object, and not by hardcoding it.